matRad/doseCalc/+DoseEngines/matRad_ParticleAnalyticalBortfeldEngine.m

```diff
@@ -324,6 +326,11 @@ function calcLateralParticleCutOff(this,cutOffLevel,~)
                 calcRange = true;
             end
 
+            if strcmp(this.cutOffMethod, 'relative')
+                warning('Relative cutoff not yet implemented for Analytical Engine. Default integral method used.')
```

Comment:
In princpile, you should use matRad_cfg.dispWarning here.
However, I think that in this case of a symmetrical centered (i.e., radial) Gaussian both methods actually arrive at the same result. You can verify this with this piece of code:

```matlab
rg = @(r) 1/(2*pi) * exp(-r.^2/2); %Radial 2D Gaussian
rg(sqrt(2) * sqrt(gammaincinv(0.995,1)))/rg(0); %Resulting relative value is exactly 1-0.995.
```

This means you can just use the same method for both here in the analytical case.